controllers/controller/workspacerouting/workspacerouting_controller.go

```diff
@@ -79,7 +79,7 @@ func (r *WorkspaceRoutingReconciler) Reconcile(req ctrl.Request) (ctrl.Result, e
 	}
 
 	if instance.Status.Phase == controllerv1alpha1.RoutingFailed {
-		return reconcile.Result{}, err
+		return reconcile.Result{}, nil
```

Comment:
Not sure if I missed something, but why are we no longer returning an error here?

Reply:
I've removed err from here because it's confusing.
This err variable is declared in another place, and if any error occurred - it's thrown much earlier.
Here it's always nil. So, I'm returning nil here to make it clearer that when routing is failing we do nothing.